libs/core/tests/unit_tests/test_tools.py

```diff
@@ -987,3 +988,33 @@ def foo(
         },
         "required": ["bar", "baz"],
     }
+
+
+def test_convert_from_runnable() -> None:
+    def f(x: List[int]) -> int:
+        """Calculate a maximum."""
+        return max(x)
+
+    def g(x: int) -> str:
```

Comment:
Most of our Runnables as defined as dicts for the first argument. Would you mind sharing an example of how that would look like?

Reply:
added